service/interpreter/globalVersioner.go

```diff
@@ -58,6 +59,10 @@ func (p *GlobalVersioner) IsAfterVersionOfTemporal26SDK() bool {
 	return p.version >= StartingVersionTemporal26SDK
 }
 
+func (p *GlobalVersioner) IsAfterVersionOfNoIwfGlobalVersionSearchAttribute() bool {
```

Comment:
actually, we don't have to strictly use this naming pattern. I think we could just rename the existing one to ```go
IsUsingGlobalVersionSearchAttribute(){
   return version >= .. && <= ...; 
}